app/lib/admin/backend.dart

```diff
@@ -802,6 +804,7 @@ class AdminBackend {
     }
 
     // delete package versions
+    // TODO: merge this with the removePackageVersion method.
```

Comment:
Consider creating an issue for this TODO

Reply:
Ah, that's actually a TODO that was implemented in the follow-up commit, will remove the line.